protocol/README.md

```diff
@@ -0,0 +1,66 @@
+<section style="height: 30vw; min-height: 15rem;
+      background: linear-gradient(#268762, #15ac7f)">
+        <div style="
+          height: 30vw;
+          min-height: 15rem;
+          background-image: url(https://raw.githubusercontent.com/ReproNim/demo-protocol/master/VoicePilot/about_the_study.svg);
+          background-position: center;
+          background-size: contain;
+          background-repeat: no-repeat">
+        </div>
+      </section>
+      <br>
+<section>
+<div class="container-fluid">
+  <h2>Welcome to the Demonstration Protocol</h2>
```

Comment:
This document should be updated to reflect the screening protocol. Also this doc should list pointers to national resources for people who need immediate help.

Reply:
@danielmlow will be updating it.